### DISCUSSION TITLE
About Pegasus Testnet Network Metamask Add Chain ID Error.

### DISCUSSION THREAD

**Discussion by Tradersoros06** [squidfunk/mkdocs-material#5397] [General]
When we try to add a metamask to the Pegasus Tesnet network manually or via chainlist, the screen error is as follows. I sent a lightlink mail for the solution, but I did not receive a response, can a helpful friend who can support or try tesnet explain the subject, thank you.

![tesnet aln](https://user-images.githubusercontent.com/129700233/233391332-08971bb2-5f29-4750-bdc5-4346586806ab.png)
![ll main](https://user-images.githubusercontent.com/129700233/233391373-d39036c4-df46-47e0-b776-5cfe27c82145.png)
![ll tesnet](https://user-images.githubusercontent.com/129700233/233391401-0b328848-af50-431b-9102-2197cadcc68a.png)

**Comment by squidfunk** [comment]
This question has nothing to do with this project.
